variables.tf

```
@@ -102,6 +102,20 @@ variable "ipv6_enabled" {
   nullable    = false
 }
 
+variable "enable_private_ipv6" {
+  type        = bool
+  description = "Set `false` to disable IPv6 addresses in private subnets"
+  default     = true
+  nullable    = false
+}
+
+variable "enable_public_ipv6" {
```

```suggestion
variable "public_ipv6_enabled" {
```

See the above comment